<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget_main</class>
 <widget class="QWidget" name="widget_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>349</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>401</width>
    <height>349</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Widget Main</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QLabel" name="lbl_patch_status_indicator">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 106, 0);</string>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_patch_status_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">Unpatched</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_contribute">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">Contribute</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_revert">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>83</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">Revert</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_patch">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>83</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">Patch</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_update_indicator">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>20</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_update_text">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>142</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string notr="true">No update available</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_commit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">Commit Edits</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_update">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_index">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">Index Game Text</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
